pkg/server/server.go

```diff
@@ -92,6 +103,15 @@ WantedBy=multi-user.target
 	return nil
 }
 
+func appendMachineConfig(mc *mcfgv1.MachineConfig) error {
```

Comment:
Could use a comment here for the future to save people going through `git blame` for why this exists, something like:

```suggestion
// appendInitialMachineConfig saves the full serialized MachineConfig that was served
// by the MCS when the node first booted.  This currently is only used as a debugging aid
// in cases where there is unexpected "drift" between the initial bootstrap MC/Ignition and the one
// computed by the cluster.
func appendInitialMachineConfig(mc *mcfgv1.MachineConfig) error {
```

Reply:
fixed